06-improve-model-performance/improvement-overview.ipynb

```diff
@@ -0,0 +1,913 @@
+{
```

Comment:
<p>Let's also show how to filter down to results w/ <code>equivalence == 0.0</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ContextualAI/examples/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>Good call, added it</p>